src/three-components/PMREMGenerator.ts

```diff
@@ -208,10 +225,12 @@ export class PMREMGenerator {
       generateMipmaps: false,
       type: UnsignedByteType,
       format: RGBEFormat,
-      encoding: RGBEEncoding
+      encoding: RGBEEncoding,
+      stencilBuffer: false
     };
     const cubeUVRenderTarget = this[$createRenderTarget](params);
-    this[$pingPongRenderTarget] = this[$createRenderTarget](params);
+    this[$pingPongRenderTarget] =
```

Comment:
This is kind of a meta-comment, but it is tricky to reason about when this gets disposed from the place where it is allocated or assigned.

Reply:
I refactored it; see if this looks better.